dill/_dill.py

```diff
@@ -1555,13 +1555,6 @@ def _get_attr(self, name):
     # stop recursive pickling
     return getattr(self, name, None) or getattr(__builtin__, name)
 
-def _dict_from_dictproxy(dictproxy):
```

Comment:
I think this is ok to delete as it's not a registered function (so no issues with stored pickles requiring it).